index.html

```diff
@@ -6,22 +6,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" href="css/style.css">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Portofilio: mobile version</title>
 </head>
 
-<body>
+<body>  
 
-<section>
+   <container id='wraper'>
```

Comment:
- [ ] I understand that this section should be a container. but `container` is not a valid html tag you can use on of the valid HTML tags such as `main` or `div`. it is crucial to follow proper HTML semantics 👍. 
Something like this would be better:
```suggestion
   <main id='wraper'>
```